ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
-
+#!/usr/bin/env/python
```

Comment:
Revert, the previous is more stable across systems and (last I checked) the recommended shebang for executable Python files. The version you added requires a Python path to exist at that location, which it likely will not.

Reply:
done